src/test/kotlin/com/igorwojda/list/countuniquevalues/challenge.kt

```diff
@@ -4,7 +4,7 @@ import org.amshove.kluent.shouldBeEqualTo
 import org.junit.jupiter.api.Test
 
 private fun countUniqueValues(list: List<Int>): Int {
-    TODO("not implemented")
+    return list.distinct().size
```

Comment:
challenge: convert the List to a different type of collection that yields the unique items in the list. Can you think which one it is?